pandas/tests/dtypes/test_dtypes.py

```diff
@@ -54,7 +55,8 @@ def test_pickle(self):
 class TestCategoricalDtype(Base):
 
     def create(self):
-        return CategoricalDtype()
+        # TODO(GH 26403): Remove when default ordered becomes False
+        return CategoricalDtype(ordered=None)
```

Comment:
if your comment is correct that this can be removed again, shouldn't it then be `ordered=False` ?

Reply:
My read is that the intention here is to test the default construction for `CategoricalDtype`, which is still currently `ordered=None`; just explicitly specified it to avoid needing to catch more warnings.

It looks like specifying `ordered=False` doesn't actually break anything though, so I could make that change now if you want.